### DISCUSSION TITLE
21-06-17 일일회고

### DISCUSSION THREAD

**Discussion by eeeesong** [d-h-k/issue-tracker#75] [회고]
오늘은 제가 올려보겠습니다😎

수료 D-8!

**Comment by eeeesong** [comment]
오늘은 잭슨과 합정에서 만나서 프로젝트 진행했습니다!
피치못할(?) 사정으로 짝 프로그램 비슷하게 진행했는데, 결과적으로 필요한 시간이었다고 느꼈습니다!

한 컴퓨터를 같이 보며 프로젝트 전반적으로 사용하는 Network Layer를 사용하기 편하게 정리하고,
리뷰어 의견에 따라 유저 정보를 싱글톤 패턴으로 바꾸는 리팩토링 작업을 했습니다.
그리고 각자 구성한 View에서 고민거리나 어려운 점들을 나눴는데
남은 시간동안 협업을 좀 더 수월하게 진행할 수 있는 자양분이 될 것 같습니다!!

다들 이번주 개발 잘 마무리하시고, 내일 데모도 화이팅입니다!

**Comment by JacksonPk** [comment]
짝 프로그래밍을 하고 싶었습니다. 롤로와 잘 협력했고 생각해보지 못한 부분을 발견해 같이 해결해 나가서 좋았습니다.
그동안 회고록을 습관적으로 까먹어서 못썼는데 앞으로 꼭 써보겠습니다.
다들 오늘도 화이팅입니다!

**Comment by GleamingStar** [comment]
회고 까먹고 업로드도 못했는데 롤로가 만들어주셨네요! 늦었지만 지금이라도 남겨보겠습니다
목요일날 제가 뭘했을까요...
이슈 Filter를 atom으로 분리했고, 이를 선택하는 로직을 편집했습니다
코쿼 나가서 코드 자체는 몇 줄 못짰네요... 동기분들이랑 많은 얘기를 나눴습니다!
